designs/administrator/future-openshift/resource-related-quick-starts/index.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Surfacing resource-related quick starts
+
+## Introduction
+Sometimes quick starts can be closely associated with a resource type, for example, a virtual machine or a pipeline. With that in mind, it became important to surface quick starts on that resource's page to improve the flow of both creating a resource and using a resource. This design demonstrates how to do so.
+
+## Design summary
+The design is broken up into scenarios:
+- [Populated resource list views](#populated-resource-list-views)
+- [Empty resource list views](#empty-resource-list-views)
+
+Whether the list view is populated or not, the general pattern of this design includes surfacing a dismissible label to make it clear that a quick start is available within the page heading.
+![general-design](img/general-design.png)
+
+The label is more like a hint and can be hidden, which would then hide **all** resource-related quick start labels for that user.
+![general-design-hide](img/general-design-hide.png)
+
+If there are multiple quick starts associated with a resource, they should be shown in a side panel, where the order logic should be as follows:
+- In progress quick starts
+- Not started quick starts
+- Do not show completed quick starts
+![side-panel](img/side-panel.png)
+
+At the end of the list, there should be a link to view all quick starts associated with that resource. The button should take the user to the quick starts catalog page that is filtered by the resource type.
+![side-panel-cta](img/side-panel-cta.png)
+
+## Empty resource list views
+If a quick start exists that will help the user create a resource type, it should be made available within the empty state of the list view.
+![empty-list](img/empty-list.png)
+
+Selecting the button will immediately start the quick start on the page.
+![empty-list-click-single](img/empty-list-click-single.png)
+
+If multiple quick starts exist that will help the user create a resource type, a side panel should list the quick starts.
+![empty-list-click-multiple](img/empty-list-click-multiple.png)
+
+Choosing to resume or start a quick start from the side panel will resume or start the quick start on that page.
+
+## Populated resource list views
+If a quick start exists that will help the user use or explore a resource type, it should be made available as a label in the page heading.
+![populated-list](img/populated-list.png)
+
+Selecting the label will immediately start the quick start on the page.
+![populated-list-click-single](img/populated-list-click-single.png)
+
+If multiple quick starts exist that will help the user use or explore a resource type, a side panel should list the quick starts.
```

Comment:
Was wondering if plural label "Quick starts available" could be done here? Or maybe if it's a pain to do singular and plural we just always do plural?

Reply:
Actually that's a good question @abigaeljamie do you have any suggestions? Whenever the label is exposed, there will be _at least_ one quick start that the user can take that connects to that resource. It's probably easiest implementation-wise to use the same term whether there's 1 or more, but wondering what you think

Reply:
What happens when the user clicks the button (where do they go)? It might make sense to have something action-oriented that we can use each time (and probably go with plural to cover all our bases). Example: "View quick start guides" or "Access quick start guides."

Reply:
@abigaeljamie We reference them as "quick starts" elsewhere in the console (the quick start catalog page, as well as the quick start card we have on our dashboard today)... knowing that, I'm not sure if we should include "guide" in there since we haven't been doing that anywhere else.

When they click the button, they will either:
1) stay on the page and a drawer will slide out that lists the quick starts (if there are multiple) --> clicking one would then start that quick start
2) immediately begin the quick start that is associated with that resource